App/src/main/java/User.java

```diff
@@ -0,0 +1,76 @@
+
+
+public class User {
+	/*For the User class the fields needed are:
```

Comment:
Try to keep the formatting of comments consistent. Especially for future readers that are just skimming through, it's nicer if the *s start on the same place in every line, etc.